experiments/integrated/ozark/conservation/ozark_conservation.jl

```diff
@@ -15,442 +15,485 @@ using ClimaLSM.Canopy
-    κ_sat_unfrozen = κ_sat_unfrozen,
-    ρc_ds = ρc_ds,
-    ν = soil_ν,
-    ν_ss_om = ν_ss_om,
-    ν_ss_quartz = ν_ss_quartz,
-    ν_ss_gravel = ν_ss_gravel,
-    hydrology_cm = vanGenuchten(; α = soil_vg_α, n = soil_vg_n),
-    K_sat = soil_K_sat,
-    S_s = soil_S_s,
-    θ_r = θ_r,
-    earth_param_set = earth_param_set,
-    z_0m = z_0m_soil,
-    z_0b = z_0b_soil,
-    emissivity = soil_ϵ,
-    PAR_albedo = soil_α_PAR,
-    NIR_albedo = soil_α_NIR,
-);
-
-soil_args = (domain = soil_domain, parameters = soil_ps)
-soil_model_type = Soil.EnergyHydrology{FT}
-
-# Soil microbes model
-soilco2_type = Soil.Biogeochemistry.SoilCO2Model{FT}
-
-soilco2_ps = SoilCO2ModelParameters{FT}(;
-    ν = soil_ν,
-    θ_a100 = θ_a100,
-    D_ref = D_ref,
-    b = b,
-    D_liq = D_liq,
-    # DAMM
-    α_sx = α_sx,
-    Ea_sx = Ea_sx,
-    kM_sx = kM_sx,
-    kM_o2 = kM_o2,
-    O2_a = O2_a,
-    D_oa = D_oa,
-    p_sx = p_sx,
-    earth_param_set = earth_param_set,
-);
-
-# soil microbes args
-Csom = (z, t) -> 5.0
-
-soilco2_top_bc = Soil.Biogeochemistry.SoilCO2StateBC((p, t) -> atmos_co2(t))
-soilco2_bot_bc = Soil.Biogeochemistry.SoilCO2StateBC((p, t) -> 0.0)
-soilco2_sources = (MicrobeProduction{FT}(),)
-
-soilco2_boundary_conditions =
-    (; top = (CO2 = soilco2_top_bc,), bottom = (CO2 = soilco2_bot_bc,))
-
-soilco2_drivers = Soil.Biogeochemistry.SoilDrivers(
-    Soil.Biogeochemistry.PrognosticMet{FT}(),
-    Soil.Biogeochemistry.PrescribedSOC{FT}(Csom),
-    atmos,
-)
-
-soilco2_args = (;
-    boundary_conditions = soilco2_boundary_conditions,
-    sources = soilco2_sources,
-    domain = soil_domain,
-    parameters = soilco2_ps,
-    drivers = soilco2_drivers,
-)
-
-# Now we set up the canopy model, which we set up by component:
-# Component Types
-canopy_component_types = (;
-    autotrophic_respiration = Canopy.AutotrophicRespirationModel{FT},
-    radiative_transfer = Canopy.TwoStreamModel{FT},
-    photosynthesis = Canopy.FarquharModel{FT},
-    conductance = Canopy.MedlynConductanceModel{FT},
-    hydraulics = Canopy.PlantHydraulicsModel{FT},
-    energy = Canopy.BigLeafEnergyModel{FT},
-)
-# Individual Component arguments
-energy_args = (parameters = Canopy.BigLeafEnergyParameters{FT}(ac_canopy),)
-
-# Set up autotrophic respiration
-autotrophic_respiration_args = (;
-    parameters = AutotrophicRespirationParameters{FT}(;
-        ne = ne,
-        ηsl = ηsl,
-        σl = σl,
-        μr = μr,
-        μs = μs,
-        f1 = f1,
-        f2 = f2,
+include(joinpath(climalsm_dir, "parameters", "create_parameters.jl"))
+
+# Define simulation times
+t0 = Float64(120 * 3600 * 24)# start mid year
+N_days = 10
+tf = t0 + Float64(3600 * 24 * N_days)
+dt = Float64(150)
+n = 1
+saveat = Array(t0:(n * dt):tf)
+
+for (float_type, tf) in ((Float32, t0 + 2 * dt), (Float64, tf))
+    # Make these global so we can use them in other ozark files
+    global FT = float_type
+    global earth_param_set = create_lsm_parameters(FT)
```

Comment:
I also don't think this setup is super clean, but since these are defined in the for loop they need to be global so the other included files can access them

Reply:
it's ok! but if we move ozark_sim inside the loop, then everything is inside the loop I think?

Reply:
even when ozark_sim is called from inside the loop, `FT` and `earth_param_set` have to be specified as global to be accessed from other files. I don't exactly understand how the scope is defined, but I get an error that `FT` is undefined from within ozark_sim if I don't make it global :/